File: server/src/main/java/com/mthwate/conk/block/BlockDirt.java
package com.mthwate.conk.block;

import com.mthwate.conk.command.Exec;

/**
 * @author mthwate
 */
@Exec
public class BlockDirt extends BlockBase {

	@Override
	public String getName() {
		return "dirt";
	}

	@Override
	public String[] getTextures() {
		return new String[] {getName(), getName() + "Side"};
	}

}

File: server/src/main/kotlin/com/mthwate/conk/block/BlockDirt.kt
package com.mthwate.conk.block

import com.mthwate.conk.command.Exec

/**
 * @author mthwate
 */
@Exec
class BlockDirt() : AbstractBlock("topsoil") {

	override val textures = arrayOf(name, name + "Side")

}
